src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -265,11 +265,21 @@ This package is an internal implementation of the .NET Core SDK and is not meant
           BeforeTargets="_GetPackageFiles;GenerateNuspec"
           DependsOnTargets="ResolveReferences">
     <ItemGroup>
-      <!-- Native files are included in _PackageFile instead because they belong in a separate package folder.r -->
+      <!-- Native files are included in _PackageFile instead because they belong in a separate package folder. -->
       <Content Include="@(ReferenceCopyLocalPaths)" PackagePath="$(NativeAssetsPackagePath)" Condition=" '%(ReferenceCopyLocalPaths.IsNativeImage)' == 'true' AND '%(ReferenceCopyLocalPaths.Extension)' != '.pdb' " />
     </ItemGroup>
   </Target>
 
+  <Target Name="_ResolveRuntimePackContent"
+          BeforeTargets="_GetPackageFiles"
+          DependsOnTargets="GenerateSharedFxDepsFile">
+    <ItemGroup>
+      <RuntimePackContent Include="$(PlatformManifestOutputPath)" PackagePath="$(ManifestsPackagePath)" />
+
+      <_PackageFiles Include="@(RuntimePackContent)" />
```

Comment:
Odd to update an internal item group of another target. Suggest using an item group `_GetPackageFiles` reads from e.g. `@(Content)` or `@(None)`.

Reply:
I was copying the logic from App.Ref.csproj. Seems like the indirection was introduced in https://github.com/aspnet/AspNetCore/commit/cc065f0055b72401a666f9eb2bb579bcc37870ce. @natemcmaster any reason for this? If not I'll just use the Content itemgroup.

Reply:
This implementation is fine, no compelling reason to use content or none item groups IMO.